crowbar_framework/app/helpers/installer/upgrades_helper.rb

```diff
@@ -74,9 +72,7 @@ def upgrade_ha_repo?
     end
 
     def upgrade_ceph_repo_needed?
-      return true if Proposal.where(barclamp: "ceph").first
-
-      false
+      !Proposal.where(barclamp: "ceph").empty?
```

Comment:
I prefer `any?` as well. It sounded a little strange to me at first, but it makes perfect sense now after reading what [where](http://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-where) does and what it [returns](http://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-any-3F).